Add support of new case types

julmud opened the pull request:
In DVD Profiler 3.9, some new case types were added (Fr4me and Handle).
This commit adds support for displaying both of those frame types
in phpDVDProfiler.

I also did some local reordering of the entries in the language
files for better readability.

Fixes #14

gjnoonan reviewed (approved):
I'd split into 2 commits. One for the reordering and oen adding the 2 new options. It will save a lot of noise later if having to do a bisect.

Other than that, approved.

julmud committed:
Reorder case types in language files

This should improve a little bit the readability of
the language files.

julmud committed:
Add support of new case types

In DVD Profiler 3.9, some new case types were added (Fr4me and Handle).
This commit adds support for displaying both of those frame types
in phpDVDProfiler.

Fixes #14
